### PR TITLE
implement `static` functions, e.g: getTVl

### PR DESCRIPTION
**Pull request by dylenfu**
The `defillama` website is famous for various `defi TVL` rankings.

How to access `defilama`:
1.fork the repo of `github.com/DefiLlama/DefiLlama-Adapters` to your own git repository;
2.add folder which named as your `defi` project into projects.
3.fetch the `defi tvl` with on chain calculation.
4.submit the pull request to the adapters and waiting for tests and merging.
5.after that, the TVL of your DeFi project will be displayed on the website.

the contract of `static` will get available tvl and the sum of miners balance from `fil-liquid`, and the interface of `getTVL` will be used in the adapter.

### CODE REVIEW

File: contracts/Static.sol [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+contract Static {
+    
+    FILLiquidInterface public _filliquid;
+
+    struct TVLData {
+        uint minerNum;
+        uint activeMinerNum;
+        uint minerBalance;
+        uint totalTVL;
+        uint availableTVL;
```
Comment (steven004) [inline contracts/Static.sol:17]:
```suggestion
        uint collaterizedMinerNum;
        uint totalLockedMinerBalance;
        uint totalFILLiquidity;
        uint availableFILLiquidity;
        uint TVL;
```

Some wording suggestion!

File: contracts/Static.sol [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            FILLiquid.BindStatusInfo memory info = allMiners[j];
+            if (!info.status.stillBound) continue;
+            data.activeMinerNum++;
+            data.minerBalance += minerBalance(info.minerId);
+        }
+
+        FILLiquid.FILLiquidInfo memory filLiquidInfo = _filliquid.getStatus();
+        data.minerNum = end;
+        data.totalTVL = filLiquidInfo.totalFIL;
+        data.availableTVL = filLiquidInfo.availableFIL;
```
Comment (steven004) [inline contracts/Static.sol:39]:
```suggestion
        for (uint j = 0; j < allMiners.length; j++) {
            FILLiquid.BindStatusInfo memory info = allMiners[j];
            if (!info.status.stillBound) continue;
            data.collaterizedMinerNum++;
            data.totalLockedMinerBalance += minerBalance(info.minerId);
        }

        FILLiquid.FILLiquidInfo memory filLiquidInfo = _filliquid.getStatus();
        data.minerNum = end;
        data.totalFILLiquidity = filLiquidInfo.totalFIL;
        data.availableFILLiquidity = filLiquidInfo.availableFIL;
        data.TVL = data.totalLockedMinerBalance + data.availableFILLiquidity;
```

**Review by steven004 — COMMENTED** [review]
My comments above are just suggestions. 
Not sure if this is. compliant with the requirement. Please feel free to fix if not.

**Comment by steven004** [comment]
LGTM.

**Review by zhiqiangxu — APPROVED** [review]

**Pull request merged**